content/docs/instrumenting/exporters.md

```diff
@@ -65,6 +65,7 @@ hosted outside of the Prometheus GitHub organization.
    * [Rsyslog exporter](https://github.com/digitalocean/rsyslog_exporter)
    * [scollector exporter](https://github.com/tgulacsi/prometheus_scollector)
    * [SMTP/Maildir MDA blackbox prober](https://github.com/cherti/mailexporter)
+   * [speedtest exporter](https://github.com/RichiH/speedtest_exporter)
```

Comment:
Seems their capitalization is "Speedtest" on http://www.speedtest.net/about, so let's capitalize the S?